org.hl7.fhir.validation/src/main/java/org/hl7/fhir/validation/special/PackageReGenerator.java

```diff
@@ -79,12 +77,17 @@ public static class TerminologyResourceEntry {
   }
 
   public static void main(String[] args) throws Exception {
+    String tmpdir = Files.createTempDirectory("repackage").toFile().getAbsolutePath();
+
     new PackageReGenerator()
-      .addPackage("hl7.fhir.us.davinci-alerts")
+      .addPackage("ch.fhir.ig.ch-core")
       .setJson(true)
+      .addIgnoreList(List.of("http://fhir.ch/ig/ch-core/StructureDefinition/ch-core-patient-epr", "http://fhir.ch/ig/ch-core/StructureDefinition/ch-core-claim", "urn:ietf:bcp:47"))
       .setOutputType(ExpansionPackageGeneratorOutputType.TGZ)
-      .setOutput("/Users/grahamegrieve/temp/vs-output.tgz")
+      .setOutput(tmpdir + "/some-output.tgz")
```

Comment:
I believe I misunderstood this in my last review; please use the `String[] args` object instead of tmpdir. We're already changing the expected behaviour for @grahamegrieve, but using `args` will ensure some consistency vs. a system dependent temp directory.

Reply:
Its now reverted. If it was my call I would delete it entirely.